compiler/rustc_error_codes/src/error_codes/E0227.md

```diff
@@ -0,0 +1,33 @@
+This error indicates that the compiler is unable to determine whether there is
+exactly one unique region in the set of derived region bounds.
+
+Example of erroneous code:
+
+```compile_fail,E0227
+trait Foo<'foo>: 'foo {}
+trait Bar<'bar>: 'bar {}
+
+trait FooBar<'foo, 'bar>: Foo<'foo> + Bar<'bar> {}
+
+struct Baz<'foo, 'bar> {
+    baz: dyn FooBar<'foo, 'bar>,
+}
+```
+
+Here, `baz` can have either `'foo` or `'bar` lifetimes.
+
+To resolve this error, provide an explicit lifetime:
+
+```rust
```

Comment:
nit: You don't need to add `rust`, it's the default. It's not a blocker for the PR though.

Reply:
It's nice to have syntax highlighting in vim while editing `.md` files.

Reply:
Like I said, not a blocker. We generally don't add it but it's fine to have it here.